### ISSUE TITLE
Idea: Group badge

### ISSUE THREAD

**Issue by rampa3** [Neos-Metaverse/NeosPublic#2448]
## Is your feature request related to a problem? Please describe.
The situation is: I am a member of a group, and our group leader wants to have a group badge, but this feature doesn't exist at the moment.

## Relevant issues
No relevant issues found.

## Describe the solution you'd like
I propose to add a group badge system in the future, where the group leader will be able to decide, if they want a group badge, that would be automatically added to badges of the group members.

## Describe alternatives you've considered
At the moment, leader has our group badge added as Patreon custom badge, while I use user badge manager from Epsilion to wear it. I only consider these alternative solutions as temporary, and would like to see an official solution.

## Additional context
None.

**Comment by shadowpanther** [comment]
While I see the benefits of this, I also want to say that one user could participate in multiple groups and if each group adds a badge, all in different styles, it would be a colorful mess. And some users might prefer to not show the groups they are in, for various reasons.

**Comment by Frooxius** [comment]
This is something we plan on adding once the Profile system is setup. To avoid badge spam, you'll have to actively choose which of the badges accessible to you (your custom badges and badges of any groups you're in) you want displayed in the world, with some limit on how many can be active at the time.

That way simply joining a group will only make the badge available for you for picking, but you won't have it displayed by default. If you do pick it, you'll have limited amount of slots that you can fill (but you can change the loadout at any time), since the number of groups you can be in is potentially quite large.

**Comment by Aerizeon** [comment]
@Frooxius Will be the API still provide a way to retrieve a complete list of badges for a given user, or will hiding a badge make it completely inaccessible?

**Comment by Frooxius** [comment]
Yes. Those badges would still be on your profile and people could see all of them there in the profile detail. It's just for showing them in world/session and on your profile in a prominent way, you'll pick the ones you want to showcase.

Not sure how much the API will change though, it'll probably get formalized better, rather than using tags as it is right now.
